packages/cli/src/cli/cogify/action.make.cog.ts

```diff
@@ -0,0 +1,158 @@
+import { Epsg, TileMatrixSet, TileMatrixSets } from '@basemaps/geo';
+import { fsa, LogConfig, LogType } from '@basemaps/shared';
+import { CommandLineAction, CommandLineIntegerParameter, CommandLineStringParameter } from '@rushstack/ts-command-line';
+import { CogStacJob, JobCreationContext } from '../../cog/cog.stac.job.js';
+import { ProjectionLoader } from '../../cog/projection.loader.js';
+import * as ulid from 'ulid';
+import { getCutline } from './cutline.js';
+import { CogJobFactory } from '../../cog/job.factory.js';
+
+function guessProjection(imageryName: string): Epsg | undefined {
```

Comment:
We shouldn't need this any more. all the imagery is "standardised"

Reply:
So the projection always be 2193 after standardised?

Reply:
the imagery will ways be valid geotiffs, this was because the imagery is not valid geotiffs.